packages/clerk-js/src/ui/foundations/opacity.ts

```diff
@@ -1,4 +1,5 @@
 export const opacity = Object.freeze({
+  hidden: '0%',
```

Comment:
Not a strong opinion held, but it doesn't feel like we need a hidden opacity within the scale, since that'd always resolve 0%.